homeassistant/components/nextbus/sensor.py

```diff
@@ -108,41 +108,21 @@ def __init__(self, client, agency, route, stop, name=None):
         self.agency = agency
         self.route = route
         self.stop = stop
-        self._custom_name = name
+        self._attr_extra_state_attributes = {}
+
         # Maybe pull a more user friendly name from the API here
-        self._name = f"{agency} {route}"
-        self._client = client
+        self._attr_name = f"{agency} {route}"
+        if name:
+            self._attr_name = name
 
-        # set up default state attributes
-        self._state = None
-        self._attributes = {}
+        self._attr_unique_id = "_".join((agency, route, stop))
```

Comment:
This would not classify as unique so I think this one has to go in the next PR when you have an entry_id to use to make it unique